examples/jira_mysql_install.pp

```diff
@@ -0,0 +1,24 @@
+node default {
+
+  class { '::mysql::server':
+    root_password    => 'strongpassword',
+  } ->
+
+  mysql::db { 'jira':
+    user     => 'jiraadm',
+    password => 'mypassword',
+    host     => 'localhost',
+    grant    => ['ALL'],
+  } ->
+
+  class { '::jira':
+    javahome    => '/opt/java/latest',
+    db          => 'mysql',
+    dbport      => '3306',
+    dbdriver    => 'com.mysql.jdbc.Driver',
+    dbtype      => 'mysql',
+  } ->
+
+  class { '::jira::facts': }
```

Comment:
To be honest for me its just semantics, I really don’t mind either way. I think include is more correct, so I'll update the example. The dependency is not required, just a bad habit where I tend to want to run things in the exact same order every time.